ptn/buttonrolebot/ui_elements/ButtonConfig.py

```diff
@@ -515,16 +545,37 @@ async def final_submit_button(self, interaction: discord.Interaction, button):
                 raise BadRequestError(e)
             except Exception as e:
                 await on_generic_error(spamchannel, interaction, e)
-
         except Exception as e:
             try:
                 raise GenericError(e)
             except Exception as e:
                 await on_generic_error(spamchannel, interaction, e)
 
+    @discord.ui.button(
```

Comment:
As above comment, I would wonder about putting this on the final page (index = 5), so the user has confirmed the details of their previous button and clicked Confirm, before being asked to add another?

Reply:
It's easily moveable if you want this to be the case, I personally think it would be better on the last page though